src/org/traccar/api/resource/UserResource.java

```diff
@@ -50,6 +53,18 @@ public Response add(User entity) throws SQLException {
         if (!Context.getPermissionsManager().isAdmin(getUserId())) {
             Context.getPermissionsManager().checkRegistration(getUserId());
             Context.getPermissionsManager().checkUserUpdate(getUserId(), new User(), entity);
+            Config config = Context.getConfig();
+            int deviceLimit = config.getInteger("default.deviceLimit");
+            int expirationDuration = config.getInteger("default.expirationDuration");
+            if (deviceLimit > 0) {
+              entity.setDeviceLimit(deviceLimit);
+            }
+            if (expirationDuration > 0) {
+              Calendar cal = Calendar.getInstance();
```

Comment:
I couldn't find a `Date` method to add days. Do you mean converting `Date` to epoch milliseconds then adding the _days_?